src/main/java/org/kainos/ea/resources/JobRoleController.java

```diff
@@ -0,0 +1,40 @@
+package org.kainos.ea.resources;
```

Comment:
Missing integration tests

Reply:
Does the ControllerTest not count as an integration test?

Reply:
Ok i see what you mean now